blog/_posts/2017-11-30-bloop-release.md

```diff
@@ -128,19 +149,27 @@ every configuration file contains basic information about the project: name, bas
 classpath, source directories, classes directory, Scala and Java options and a few more.
 
 `bloop` comes with an sbt plugin that allows you to generate these
-configuration files with the execution of a command. When the command is done,
-you can run your build with `bloop` and close the sbt instance. You will only
-need to use sbt when you have changed the architecture of your build (to
-generate the configuration files again) or for tasks unrelated to compilation
-and test execution.
+configuration files from your existing, functional sbt build. When the generation is done,
+you will have a `.bloop-config` in your base directory. With this configuration,
```

Comment:
"you will have a new directory called `.bloop-config` in the base directory of your sbt project" maybe?